javascript-linting-rules.md

```diff
@@ -0,0 +1,54 @@
+# Javascript Linting Rules
+
+The following rules should be applied to all of our javascript projects.
+
+_A project may use additional rules to ones listed here, but these would not be part of the current standard and such rules are subject to change in the future._
+
+## Current Rules
+
+These are stored in the [.eslintrc](https://github.com/holidayextras/culture/.eslintrc) in this repository.
```

Comment:
Would it make more sense to link to the raw file here instead so it can be easily downloaded?